torba/torba/server/util.py

```diff
@@ -182,7 +182,7 @@ class LogicalFile:
     """A logical binary file split across several separate files on disk."""
 
     def __init__(self, prefix, digits, file_size):
-        digit_fmt = '{' + ':0{:d}d'.format(digits) + '}'
+        digit_fmt = '{' + f':0{digits:d}d' + '}'
```

Comment:
Can the `'{'` and `'}'` go inside of the f-string?

Reply:
Yes, updated. In f-strings, curly braces are escaped by doubling them: https://stackoverflow.com/a/5466478/724176.